torchvision/prototype/datapoints/_bounding_box.py

```diff
@@ -78,7 +78,7 @@ def resize(  # type: ignore[override]
         size: List[int],
         interpolation: InterpolationMode = InterpolationMode.BILINEAR,
         max_size: Optional[int] = None,
-        antialias: Optional[bool] = None,
+        antialias: Optional[Union[str, bool]] = "warn",
```

Comment:
Technically, it is not Optional anymore as default is "warn" and not None
```suggestion
        antialias: Union[str, bool] = "warn",
```

Reply:
Honestly, I have no clue what's the difference between `Optional[bool]` and `Union[NoneType, bool]`. @pmeier any thoughts?

Reply:
I think, `Optional[X]` is `Union[X, None]`. Anyway, it is a nit, we can ignore that.